pkg/framework/aries/api/vdri/vdri.go

```diff
@@ -0,0 +1,124 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vdri
+
+import (
+	"errors"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+)
+
+// ErrNotFound is returned when a DID resolver does not find the DID.
+var ErrNotFound = errors.New("DID not found")
+
+// Registry vdri registry
+type Registry interface {
+	Resolve(did string, opts ...ResolveOpts) (*did.Doc, error)
+	Store(doc *did.Doc) error
+	Create(method string, opts ...DocOpts) (*did.Doc, error)
+}
+
+// VDRI verifiable data registry interface
+type VDRI interface {
```

Comment:
Question: Should VDRI implement closer interface? to release/dispose underlying store or other resources?

Reply:
Yes I will implement it in separate PR